### ISSUE TITLE
Rocket.addFins() does not support airfoil lift coefficient data with Cl=0

### ISSUE THREAD

**Issue by giovaniceotto** [Projeto-Jupiter/RocketPy#54]
**Use case**
Try to add fins to a rocket object using the airfoil option:

```python
FinSet = TestRocket.addFins(
    n=3, 
    rootChord=0.077, 
    tipChord=47/1000, 
    span=0.104,
    distanceToCM=-0.99586,
    airfoil='NACA0012_Cl.csv'
)
```

Contents of 'NACA0012_Cl.csv' are:

```csv
0, 6.86E-09
0.200712864, 1.05E+00
1.560796326, 0.0
```

**Expected result**
Fins are added properly with no errors  or warnings.

**What happens instead**

```
RuntimeWarning: divide by zero encountered in double_scalars FD = 2 * np.pi * AR / (cn *np.cos(gamac))
RuntimeWarning: invalid value encountered in double_scalars Cnalfa1 = cn * FD * (Af/self.area) * np.cos(gamac) / (2 + FD * ( 1 + (4/FD**2) )**0.5)
```

Furthermore, the resulting `cldata` created has a **nan** _(not a number)_ everywhere a zero is expected.

```python
[[0.00000000e+00 5.18408620e-09]
 [2.00712864e-01 7.19089724e-01]
 [1.56079633e+00            nan]]
```

I believe the error from originates from calling `cnalfa1` with `cn=0.0` in line 610 of the _Rocket.py_ file.

**Applies to versions:**
Master branch only, no releases yet.

**Comment by brunosorban** [comment]
The bug is caused by an error in the implementation of the lift coefficient calculation. This bug distorts the static margin value. A solution is already in progress, but the "airfoil" option should not be used until the  bug is fixed.

**Comment by giovaniceotto** [comment]
Great. Let me know if you need anything.

**Comment by Gui-FernandesBR** [comment]
@Projeto-Jupiter/back-end

**Comment by Gui-FernandesBR** [comment]
@lucasfourier please take a look

**Comment by Gui-FernandesBR** [comment]
I believe we can close this one based on #122 awesome implementation,

Many thanks to @MateusStano who did that. Let's rock!

**Comment by giovaniceotto** [comment]
I don't believe this is the case.

**Comment by Gui-FernandesBR** [comment]
Fixed by @MateusStano on #140 , wait for release v0.10.0

awesome work!!
